openslides_backend/models/motion_category.py

```diff
@@ -0,0 +1,40 @@
+from ..shared.patterns import Collection
+from . import fields
+from .base import Model
+
+
+class MotionCategory(Model):
+    """
+    Model for motion category.
+
+    Reverse fields:
+    - child_ids: (motion_category/parent_id)[]
+    - motion_ids: (motion/category_id)[]
+    """
+
+    collection = Collection("motion_category")
+    verbose_name = "motion_category"
+
+    id = fields.IdField(description="The id of this motion category.")
+    meeting_id = fields.RequiredForeignKeyField(
+        description="The id of the meeting of this motion_category.",
+        to=Collection("meeting"),
+        related_name="category_ids",
+    )
+    name = fields.RequiredCharField(
+        description="The name of this motion category.", maxLength=255
+    )
+    prefix = fields.RequiredCharField(
+        description="The prefix of this motion category.", maxLength=32
+    )
+    weight = fields.PositiveIntegerField(
```

Comment:
Weight can be negativ and we need a default value in create action, see my comment here: https://github.com/OpenSlides/openslides-backend/pull/94

Reply:
Done. Changed weight into an IntegerField.